### PR TITLE
Screen Orientation: add requestFullscreen to final orientation-reading.html test

### PR DESCRIPTION
**Pull request by Johanna-hub**

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]

**Comment by foolip** [comment]
@marcoscaceres another that was left unmerged here.

**Comment by marcoscaceres** [comment]
Re-triggering the checks, as one got stuck.

**Comment by foolip** [comment]
For Taskcluster closing and reopening is unfortunately not always enough, as the config is taken from the PR branch head, which might be very out of date or even not exist for old enough PRs. There's probably a bug report that @jgraham could point you to if you want to bug the Taskcluster team about it :)

File: screen-orientation/orientation-reading.html [hunk 1]
```diff
@@ -98,7 +98,8 @@
   assert_equals(screen.orientation.angle, orientation.angle);
   assert_not_equals(screen.orientation.type, orientationType);
   assert_not_equals(screen.orientation.angle, orientationAngle);
-  screen.orientation.unlock();
+  // currently doesn't return a promise, makes it less racy against exitFullScreen
```
Comment (foolip) [inline screen-orientation/orientation-reading.html]: Lint is failing on trailing whitespace somewhere in this file:
https://tools.taskcluster.net/groups/XbyvR3hiRAqnWqT2gijfEw/tasks/J-yAX1h0QvGxIjqh3j59_g/runs/0/logs/public%2Flogs%2Flive.log

**Pull request merged**